### PR TITLE
Scala 2.10.4

### PR DESCRIPTION
**Pull request by gkossakowski**

### CODE REVIEW

**Comment by gkossakowski** [comment]
Review by @adriaanm

File: hand-written.md [hunk 1]
```diff
@@ -13,12 +12,12 @@ were merged on [GitHub](https://github.com/scala/scala).
...
 
 ### Known Issues
-Before reporting a bug, please have a look at these [known issues](https://issues.scala-lang.org/issues/?filter=12417).
+Before reporting a bug, please have a look at these [known issues](https://issues.scala-lang.org/issues/?filter=12503).
 
 ### Scala IDE for Eclipse
 The Scala IDE with this release built right in is available through the following update-site:
 
-* [for Eclipse 4.2/4.3 (Juno/Kepler)](http://download.scala-ide.org/sdk/e38/scala210/dev/site/)
+* [for Eclipse 4.2/4.3 (Juno/Kepler)](http://download.scala-ide.org/sdk/helium/e38/scala210/stable/site)
```
Comment (gkossakowski) [inline hand-written.md:20]: @dragos: Is this correct URL for Scala IDE update site?

Comment (dragos) [inline hand-written.md:20]: It will be once it's built.

Comment (gkossakowski) [inline hand-written.md:20]: Yes, changes from this PR will be used as basis for PR to scala-lang. I'll hold it until IDE is published.

**Comment by adriaanm** [comment]
There are actually 33 fixed tickets, not 31. I'd suggest using this link: https://issues.scala-lang.org/issues/?jql=project%20%3D%20SI%20AND%20resolution%20%3D%20Fixed%20AND%20fixVersion%20in%20("Scala%202.10.4-RC2"%2C%20"Scala%202.10.4-RC1"%2C%20"Scala%202.10.4-RC3"%2C%20"Scala%202.10.4")%20ORDER%20BY%20priority%20DESC

**Comment by adriaanm** [comment]
There were three PRs assigned to the milestone 2.10.4-RC1 but not actually merged. So, it should be '36 RC1 pull requests'.

There's a [script](https://github.com/adriaanm/binfu/blob/master/hubfu.sh) for that:

```
$ gh_pr_closed_unmerged_milestone 22
To clear the milestone of unmerged but closed PRs for milestone 22, execute the following:
NOTE: this only looks at the first 100 PRs. Specify a page number as the second argument to look past that.
gh_pr_milestone_clear 2971
gh_pr_milestone_clear 2903
gh_pr_milestone_clear 2902
```

**Comment by adriaanm** [comment]
I would like to automate this part of the release notes, but I think that will be a job for the next infrastructure dude ;-)

**Comment by gkossakowski** [comment]
Documenting it would be a first step. I'll fix the PR and update the README.

**Comment by adriaanm** [comment]
Yes, documenting would be good. I've mentioned these scripts a couple of times in my status updates I believe, and they are part of the [2.11.0 release checklist](https://docs.google.com/a/typesafe.com/spreadsheet/ccc?key=0AvTrlTBvTpWBdFBRazJJNmhnaUdCaV9WcE1QSXl1YkE&usp=drive_web#gid=6).

**Comment by gkossakowski** [comment]
I was looking at the [2.10 release checklist](https://docs.google.com/a/typesafe.com/spreadsheet/ccc?key=0AvTrlTBvTpWBdFBRazJJNmhnaUdCaV9WcE1QSXl1YkE&usp=drive_web#gid=2) where they are not mentioned. In any case, I'll make sure we know about them when releasing 2.10 as well.

**Comment by adriaanm** [comment]
I didn't mean to blame you for not having discovered the scripts, just reminding/telling you they are there.

**Comment by gkossakowski** [comment]
@adriaanm: I fixed both number of closed tickets and number of merged PRs. Thanks for great diligence!

I left the link to JIRA filter instead of JQL for consistency reasons: all 2.10.x releases used filters. However, I agree we should use JQL for 2.11.x releases as you've been doing.

Review again, please.

**Comment by adriaanm** [comment]
LGTM, thanks for fixing my nitpicks

File: hand-written.md (hunk 1 above)
Comment (adriaanm) [inline hand-written.md:20]: @dragos, @skyluc, I got a message that people are still getting RC2 from the update site and went to the above url, which has a last modified date of Jan 13

Comment (skyluc) [inline hand-written.md:20]: Yes, it is not the URL i gave in my 'we're ready' message. This is our stable/stable location with currently Scala IDE 3.0.2 on Scala 2.10.3.
The correct location is `http://download.scala-ide.org/sdk/helium/e38/scala210/`**`dev`**`/site`, which contains Scala IDE 3.0.3-rc3 on Scala 2.10.4.
We are working on Scala IDE 3.0.3-vfinal on 2.10.4, which will be at the location you used when done.

Comment (adriaanm) [inline hand-written.md:20]: Ok, thanks for clarifying. So, should we update the release notes until 3.0.3-vfinal is available?

Comment (skyluc) [inline hand-written.md:20]: Yes. It is most likely what the user wants.

Comment (adriaanm) [inline hand-written.md:20]: Fixed in https://github.com/scala/scala-lang/commit/2667a3a517c79af442b71ce8371e0542dfad556f

Comment (gkossakowski) [inline hand-written.md:20]: Thanks guys. I really think I'll get this right the third time... :-)

**Pull request merged**